projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.integration.spec.ts

```diff
@@ -52,10 +60,10 @@ describe('IgxHierarchicalGrid Integration #hGrid', () => {
 
             expect(document.querySelectorAll('igx-grid-header').length).toEqual(3);
 
-            const firstRow = hierarchicalGrid.dataRowList.toArray()[0];
```

Comment:
You can use GridFunctions.getColumnHeaders(fixture),
instead of document.querySelectorAll('igx-grid-header')